src/test/java/CliDisplayTest.java

```diff
@@ -0,0 +1,38 @@
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.*;
+
+import static org.junit.Assert.assertEquals;
+
+public class CliDisplayTest {
+
+    private final ByteArrayOutputStream output = new ByteArrayOutputStream();
+    private final ByteArrayInputStream input = new ByteArrayInputStream("string data\n".getBytes());
+
+    @Before
+    public void beginIOStreams() {
+        System.setOut(new PrintStream(output));
+        System.setIn(input);
+    }
+
+    @After
+    public void wipeStreams() {
+        System.setOut(null);
```

Comment:
Any `System.out.print` calls from here might blow up by `NullPointerException` or might not print anything to standard output...

Not sure if JUnit handles that well and will not be a problem.

If it is a problem, `System.out` will give you the current output stream. That way you can save it in the `@Before` and restore it here.